### DISCUSSION TITLE
Welcome to billboard.js Discussions!

### DISCUSSION THREAD

**Discussion by netil** [naver/billboard.js#2002] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GDFaber** [comment]
Hi everybody, thank you for inviting me to this project 👋

I'm a newbie to billboard.js but have got some knowledge about C3.js which is helping me a lot. For a starter, I'd be happy to tackle some of the smaller bits here and there, while getting a better understanding of the library.

Looking forward to working together on this beautiful project. 😄
